doc/nrf/libraries/others/profiler.rst

```diff
@@ -13,129 +13,194 @@ The output is provided using RTT and can be visualized in a custom Python backen
 
-  Connects to the device using RTT, receives profiling data, and saves it to files.
-  As command-line arguments, provide the time for collecting data (in seconds) and a dataset name.
+  In this command, ``5`` is the time value for collecting data and ``test1`` is the dataset name.
+* :file:`plot_from_files.py` - This script plots events from the dataset that is provided as the command-line argument.
+  For example:
 
-* ``python3 plot_from_files.py test1``
+  .. parsed-literal::
+     :class: highlight
 
-  Plots events from the dataset that is provided as the command-line argument.
+     python3 plot_from_files.py test1
 
-* ``python3 real_time_plot.py test1``
+  In this command, ``test1`` is the dataset name.
+* :file:`real_time_plot.py` - This script connects to the device using RTT, plots data in real-time, and saves the data.
+  When running the script from the command line, provide the dataset name.
+  For example:
 
-  Connects to the device using RTT, plots data in real-time, and saves the data.
-  As command line arguments, provide a dataset name.
+  .. parsed-literal::
+     :class: highlight
 
-* ``python3 merge_data.py test_p sync_event_p test_c sync_event_c test_merged``
+     python3 real_time_plot.py test1
 
-  Combines data from test_p and test_c datasets into one dataset (test_merged).
-  Provides clock drift compensation based on synchronization events: sync_event_p and sync_event_c.
+* :file:`merge_data.py` - This script combines data from ``test_p`` and ``test_c`` datasets into one dataset ``test_merged``.
+  It also provides clock drift compensation based on the synchronization events: ``sync_event_p`` and ``sync_event_c``.
   This enables you to observe times between events for the two connected devices.
-  As command-line arguments, provide names of events used for synchronization for a Peripheral (sync_event_p) and a Central (sync_event_c), as well as names of datasets for: the Peripheral (test_p), the Central (test_c), and the merge result (test_merged).
+  As command-line arguments, provide names of events used for synchronization for a Peripheral (``sync_event_p``) and a Central (``sync_event_c``), as well as names of datasets for the Peripheral (``test_p``), the Central (``test_c``), and the merge result (``test_merged``).
+  For example:
+
+  .. parsed-literal::
+     :class: highlight
+
+     python3 merge_data.py test_p sync_event_p test_c sync_event_c test_merged
+
 
-Visualization
--------------
+Running the backend
+===================
 
-When running ``plot_from_files.py`` or ``real_time_plot.py``, the profiled events are visualized in a GUI window.
+To enable and run the custom backend, complete the following steps:
 
-When displaying Event Manager events, submissions are marked as dots.
-Processing of the events is displayed as rectangles, visualizing the processing time.
+1. Connect device to the computer. For more information how to configure device see :ref:`profiler_configuration`.
```

Comment:
```suggestion
1. Connect device to the computer.
2. Complete the configuration steps for use with either custom events or Event Manager, as described in the :ref:`profiler_configuration` section.
```
What do you mean exactly here? Because with how the second sentence looks like now, it literally suggests to check Configuration about how to connect the device.

Also, one line, one sentence is our rule.